var/spack/repos/builtin/packages/py-alphafold/package.py

```diff
@@ -17,31 +17,40 @@ class PyAlphafold(PythonPackage, CudaPackage):
     url = "https://github.com/deepmind/alphafold/archive/refs/tags/v2.1.1.tar.gz"
     maintainers = ["aweits"]
 
+    version("2.2.4", sha256="8d756e16f6dc7897331d834aade8493820d0ff6a03bf60ce511bac4756c1b1e8")
     version("2.1.1", sha256="1adb6e213ba9ac321fc1acb1c563ba9b4fc054c1cebe1191bc0e2aaa671dadf7")
 
     conflicts("platform=darwin", msg="alphafold is only supported on Linux")
 
+    # lots of hints on versions and patching taken from docker/Dockerfile
+    # and requirements.txt
```

Comment:
I'm not sure how much I trust these "hints". The docker file just so happens to use Python 3.7, but that doesn't suggest to me that that's the minimum version that works. Similarly with `requirements.txt`, they might just pin CI to those versions for stability reasons. I do the same thing in other projects, although it's the latest version, not the minimum version. If you want to use `requirements.txt` for minimums, I'm fine with it since you're maintaining this recipe, but it isn't what I would personally do.